docs/sources/guides/whats-new-in-v6-6.md

```diff
@@ -201,3 +201,22 @@ See [upgrade notes]({{< relref "../installation/upgrading/#upgrading-to-v6-6" >}
 ## Changelog
 
 Check out [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md) for a complete list of new features, changes, and bug fixes.
+
+## Notice about upcoming changes in backendSrv for plugin authors
+
+In our mission to migrate away from AngularJS to React we have removed all AngularJS dependencies in the core data retrieval service `backendSrv`. This change is already in master and will be introduced in the next Grafana release.
```

Comment:
maybe you need to state specifically in which release it will be introduced

Reply:
@papagian since we are not sure about 6.7 (if there will be 6.7) we sort of cannot do this

Reply:
I see, what about:
`This change is already in master and will be introduced in the next Grafana release after 6.6.0-stable`?
I think it's a bit ambiguous as it.

Reply:
Since these are 6.6 release notes/changelog, I don't think this is needed. What id needed though, is the info that it's the next `major` release, adding suggestion.